functions/federated_aws_rp/app.py

```diff
@@ -322,6 +324,8 @@ def lambda_handler(event: dict, context: dict) -> dict:
                         'body': 'Invalid POST data'}
 
                 store['role_arn'] = parsed_body['arn']
+                store['role_account_alias'] = parsed_body['alias']
+                store['role_name'] = parsed_body['role']
```

Comment:
In general I prefer `.update()` when doing things like this, but that's a minor stylistic nit.

Reply:
Sure thing, fixed in 313e6c7